lib/ansible/utils/helpers.py

```diff
@@ -20,8 +20,18 @@
 __metaclass__ = type
 
 from ansible.module_utils.six import string_types
+from ansible.module_utils._text import to_native
 
 
+class Chameleon(object):
```

Comment:
These (Chameleon, data_object_shim) are kind of unusual classes and method. So the comments/commit msgs need to explain why they are needed and how they accomplish that and why normal approaches are not applicable.